exporters/variables-scss/src/content/formatters/__tests__/cssFormatter.test.ts

```diff
@@ -0,0 +1,19 @@
+import { formatCSS } from '../cssFormatter';
+
+const mockedCSS = `$my-var: (
```

Comment:
I think it would be better to load this from a file. So you will have a `__fixtures__` directory with something like `unformatted.css` and `formatted.css` and you just load them.